_posts/2023-08-23-cluster-metrics-monitors-blog.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title:  Monitoring your cluster with cluster metrics monitors
+authors:
+  - hurneyt
+date: 2023-08-23
+categories:
+  - technical-posts
+meta_keywords: 
+meta_description: 
+---
+
+In OpenSearch version 1.3, we added cluster metrics monitors to the Alerting plugin. Cluster metrics monitors allow you to execute popular CAT and cluster API calls against the local cluster and generate alerts based on the metrics reported by those APIs. Cluster metrics monitors support calling the following APIs:
+
+1. [_cluster/health](https://opensearch.org/docs/2.9/api-reference/cluster-health/)
+2. [_cluster/stats](https://opensearch.org/docs/2.9/api-reference/cluster-stats/)
+3. [_cluster/settings](https://opensearch.org/docs/2.9/api-reference/cluster-settings/)
+4. [_nodes/stats](https://opensearch.org/docs/2.9/opensearch/popular-api/#get-node-statistics)
+5. [_cat/indices](https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-indices/) - new with v2.9.0!
+6. [_cat/pending_tasks](https://opensearch.org/docs/2.9/api-reference/cat/cat-pending-tasks/)
+7. [_cat/recovery](https://opensearch.org/docs/2.9/api-reference/cat/cat-recovery/)
+8. [_cat/shards](https://opensearch.org/docs/latest/opensearch/rest-api/cat/cat-shards/) - new with v2.9.0!
+9. [_cat/snapshots](https://opensearch.org/docs/2.9/api-reference/cat/cat-snapshots/)
+10. [_cat/tasks](https://opensearch.org/docs/2.9/api-reference/cat/cat-tasks/)
+
+
+In this blog post, we’ll walk through two common use cases for this monitor type: [monitoring CPU utilization](#monitoring-cluster-cpu-utilization) and [monitoring JVM memory pressure](#monitoring-cluster-jvm-memory-pressure). Additionally, we’ll present use cases for the newly supported APIs [`_cat/indices`](#_catindices) and [`_cat/shards`](#_catshards). 
+
+## Monitoring cluster CPU utilization
+
+By creating a cluster metrics monitor that calls the `_cluster/stats` API, you can receive alerts when the CPU utilization of the cluster reaches a certain threshold. To generate an alert when the CPU utilization reaches or exceeds 60%, follow these steps:
+
+1. From the top menu, select **OpenSearch Plugins** > **Alerting**.
+1. On the **Monitors** tab, select **Create monitor**.
+1. Select the **Per cluster metrics monitor** option. For demonstration purposes, you’ll configure this monitor to run every minute by leaving the default values under **Schedule**.
```

Comment:
I noticed that the third steps for the CPU and JVM examples don't include the `...monitor-type.png` screenshots. Those are very basic steps that basically just involve selecting the monitor type, so it may not be necessary to display those screenshots. I noticed that the `cluster-metrics-v29-cpu1-monitor-type.png` was attached to this PR though, so I just wanted to clarify whether we plan to leave it out of the blog content

Reply:
I think the instructions are self-explanatory enough that they do not necessitate a screenshot for that particular step :)